lib/s3-s3.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * AWS.S3-like behavior with failover to a secondary S3 location
+ */
+
+'use strict';
+
+// Used in init() below, this function mimics AWS.S3
+var s3 = function (primaryS3, secondaryS3) {
```

Comment:
Answering my own question.  Because by remaining AWS version agnostic it's easier for consumers to bring their own version.